src/Dapi/Types/EnrichedTransaction.cs

```diff
@@ -0,0 +1,38 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Dapi.Types {
+    public class EnrichedTransaction {
+        public float amount { get; }
+        public string date { get; }
+        public TransactionType type { get; }
```

Comment:
Use the `TransactionType` from the `TransactionType.cs` file.

Reply:
This field is being assigned a value of the wrong type, in the constructor below.